### DISCUSSION TITLE
小程序的订阅消息中，可以结合腾讯IM实现微信的弹窗通知吗？

### DISCUSSION THREAD

**Discussion by rakuchyan** [w7corp/easywechat#2710] [Q&A]
![image](https://github.com/w7corp/easywechat/assets/6279689/b74ca85a-022f-4c6f-93e0-48d23a53d7d0)
如图这样的功能。https://cloud.tencent.com/document/product/269/79050

**Comment by overtrue** [comment]
这里只是对微信接口的封装，其他平台需要自行实现哈
